docs/conceptual/snippets/fsarrays/snippet115.fs

```diff
@@ -0,0 +1,6 @@
+let array1 = [| 1 .. 100 |]
+let printArray anarray = Array.iter (printf "%A ") anarray; printfn ""
```

Comment:
Suggested change:

```fs
let printArray anarray =
    anarray |> Array.iter (printf "%A ")
    printfn ""
```

Generally avoiding semicolons is a good thing for samples, as they're hard for people to catch when reading.